docs/users.rst

```diff
@@ -935,7 +935,8 @@ tagging the resulting image in the registry:
 
   - ``{version}`` (the ``version`` label)
   - ``latest``
-  - any additional tags named in the ``additional-tags`` file
+  - any additional tags named in the ``additional-tags`` file (this file is deprecated,
```

Comment:
Maybe make this more visible?

`....(DEPRECATED: use tags ....)`

Reply:
changed